elasticsearch/templates/statefulset.yaml

```diff
@@ -315,8 +315,10 @@ spec:
             value: "{{ .Values.clusterName }}"
           - name: network.host
             value: "{{ .Values.networkHost }}"
+          {{- if .Values.esJavaOpts  }}
```

Comment:
It looks like there is a test asserting the default ES_JAVA_OPTS option that will need updated https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+template-testing/1218/CHART=elasticsearch,label=docker&&virtual/console